src/compas_fab/ghpython/components/assembly_info.py

```diff
@@ -0,0 +1,22 @@
+import GhPython
+import System
+
+assembly_name = "COMPAS FAB"
+assembly_version = "0.0.1"
```

Comment:
maybe best here would be to use the same string we use for the overall `compas_fab` version. The way to do that is to 1) use the current version here, and 2) add this file and search/replace condition to the `.bumpversion.cfg` file on the root of the repo.

Reply:
I changed bumpversion and added a few lines in 'CONTRIBUTING.rst'. But would need some help on adding a task and the naming of that task.
I created a file named install.py in src\compas_fab\ghpython\components\ that should be executed.